src/opm/input/eclipse/EclipseState/Runspec.cpp

```diff
@@ -344,9 +344,18 @@ EclHysterConfig::EclHysterConfig(const Opm::Deck& deck)
         if (!activeHyst)
 	      return;
 
-        if (!deck.hasKeyword("EHYSTR"))
-            throw std::runtime_error("Enabling hysteresis via the HYST parameter for SATOPTS requires the "
-                                     "presence of the EHYSTR keyword");
+        if (!deck.hasKeyword("EHYSTR")) {
+            std::string msg = "Hysteresis is enabled via the HYST parameter for SATOPTS, but the EHYSTR " 
+                              "keyword is not present in the deck. \n"
+                              "Default values are used for the EHYSTR keyword.";
+
+            OpmLog::warning(msg);
+            curvatureCapPrsValue = 0.1;
```

Comment:
these defaults should and are encoded as the default member var values (excluding the model indicators).

Reply:
I just thought re-setting them here made it more clear. But I can remove it. Then the commit will just change the throw to a warning.

Reply:
i think that's better, if we want to change defaults we then only have to change in one spot. however, note that you still have to set krHystMod and pcHystMod.